feature-libs/product/bulk-pricing/components/bulk-pricing-table/bulk-pricing-table.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { BulkPricingService } from '@spartacus/product/bulk-pricing/core';
+import { RoutingService } from '@spartacus/core';
+import { BulkPrice } from '@spartacus/product/bulk-pricing/core';
+import { Observable } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+
+@Component({
+  selector: 'cx-bulk-pricing-table',
+  templateUrl: './bulk-pricing-table.component.html',
+})
+export class BulkPricingTableComponent implements OnInit {
```

Comment:
This class should be exported into public api

Reply:
done